webpack/components/Table/TableWrapper.js

```diff
@@ -164,6 +189,15 @@ TableWrapper.propTypes = {
     PropTypes.bool,
   ])),
   activeFilters: PropTypes.bool,
+  displaySelectAllCheckbox: PropTypes.bool,
+  selectedCount: PropTypes.number,
+  modelName: PropTypes.string,
+  modelNamePlural: PropTypes.string,
+  selectAll: PropTypes.func,
+  selectNone: PropTypes.func,
+  selectPage: PropTypes.func,
+  areAllRowsSelected: PropTypes.func,
+  areAllRowsOnPageSelected: PropTypes.func,
```

Comment:
If all of these are only used by the 'Select all checkbox" then we should likely have a selectAllCheckBoxProps instead to clean this up a bit, the list of props on this file is getting messy.